Makefile

```diff
@@ -75,12 +75,12 @@ vet:
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
-# Build the docker image
-docker-build: unittest
+# Build the container image
+container-build: unittest
 	docker build . -t ${IMG}
```

Comment:
A side note, let's make sure this works with `podman` also, if it's available on the environment 
We're doing it already in the test container build part